src/neo-vm/EvaluationStack.cs

```diff
@@ -89,6 +89,17 @@ internal bool Reverse(int n)
             return true;
         }
 
+        public bool TryPeek<T>(out T item) where T : StackItem
```

Comment:
You don't want to add the `index=0` as argument?

Reply:
The argument is not used in `TryPeek()`.